resources/js/features/games/components/TopAchieversMainRoot/TopAchieversList/TopAchieversList.tsx

```diff
@@ -0,0 +1,85 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  BaseTable,
+  BaseTableBody,
+  BaseTableCell,
+  BaseTableHead,
+  BaseTableHeader,
+  BaseTableRow,
+} from '@/common/components/+vendor/BaseTable';
+import { PlayerBadgeIndicator } from '@/common/components/PlayerBadgeIndicator';
+import { PlayerBadgeLabel } from '@/common/components/PlayerBadgeLabel';
+import { UserAvatar } from '@/common/components/UserAvatar';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { AwardType } from '@/common/utils/generatedAppConstants';
+import { formatDate } from '@/common/utils/l10n/formatDate';
+import { cn } from '@/utils/cn';
+
+export const TopAchieversListContainerTestId = 'top-achievers-list';
```

Comment:
nit: Conventionally this should be either camelCased or SCREAMING_SNAKE_CASED.

Thankfully, given this is TS, Rename Symbol works really well.
![Screenshot 2024-11-23 at 6 33 18 PM](https://github.com/user-attachments/assets/c0fe6586-48c8-441c-9bc3-f563978a94e0)

Reply:
Based on https://github.com/RetroAchievements/RAWeb/blob/31b93029cf412c817d123e308948a8e4b7925179/resources/js/features/games/components/HashesMainRoot/HashesList/HashesList.tsx#L8

Removed.